README.md

```diff
@@ -19,437 +19,14 @@ docker-compose up
 ```
 
 Now the qiskit-service is available on http://localhost:5013/.
-
-## Transpilation Request
-Send implementation, input, QPU information, and your IBM Quantum Experience token to the API to get analyzed properties of the transpiled circuit and the transpiled OpenQASM circuit itself.
-*Note*: ``token`` should either be in ``input-params`` or extra. Both variants are combined here for illustration purposes.
-`POST /qiskit-service/api/v1.0/transpile`  
-*Note*: ``url``, ``hub``, ``group``, ``project`` are optional such that otherwise the standard values are used.
-
-#### Transpilation via URL
-```
-{  
-    "impl-url": "URL-OF-IMPLEMENTATION",
-    "impl-language": "Qiskit"/"OpenQASM",
-    "qpu-name": "NAME-OF-QPU",
-    "input-params": {
-        "PARAM-NAME-1": {
-            "rawValue": "YOUR-VALUE-1",
-            "type": "Integer"
-        },
-        "PARAM-NAME-2": {
-            "rawValue": "YOUR-VALUE-2",
-            "type": "String"
-        },
-        ...
-        "token": {
-            "rawValue": "YOUR-IBMQ-TOKEN",
-            "type": "Unknown"
-        }
-        "url": {
-            "rawValue": "YOUR-IBMQ-AUTHENTICATION-URL",
-            "type": "Unknown"
-        }
-        "hub": {
```

Comment:
think about how the structure of such params can be described with their rawValues and types